### DISCUSSION TITLE
Online Student don't have a actual student Id card

### DISCUSSION THREAD

**Discussion by CubeCoding7** [community/community#66527] [GitHub Education]
### Hi everyone,

Hi, I was just wondering what to do if I am an online student at my college since I don't have a physical card to take a picture of.

**Comment by arafajs** [comment]
You can submit another educational ID that can prove that you are a student.

**Comment by Mrt009** [comment]
Yes you can do so either by submitting your admission receipt or any similar document which mention that you are the student of that particular institute.
